test/test_gaussian.py

```diff
@@ -207,17 +207,17 @@ def test_smoke(expr, expected_type):
 
 
 @pytest.mark.parametrize('int_inputs', [
-    {},
-    {'i': bint(2)},
-    {'i': bint(2), 'j': bint(3)},
+    OrderedDict(),
+    OrderedDict([('i', bint(2))]),
+    OrderedDict([('i', bint(2)), ('j', bint(3))]),
 ], ids=id_from_inputs)
 @pytest.mark.parametrize('real_inputs', [
-    {'x': reals()},
-    {'x': reals(4)},
-    {'x': reals(2, 3)},
-    {'x': reals(), 'y': reals()},
-    {'x': reals(2), 'y': reals(3)},
-    {'x': reals(4), 'y': reals(2, 3), 'z': reals()},
+    OrderedDict([('x', reals())]),
+    OrderedDict([('x', reals(4))]),
+    OrderedDict([('x', reals(2, 3))]),
+    OrderedDict([('x', reals()), ('y', reals())]),
+    OrderedDict([('x', reals(2)), ('y', reals(3))]),
+    OrderedDict([('x', reals(4)), ('y', reals(2, 3)), ('z', reals())]),
```

Comment:
nit: I think we can revert these changes. I had set this test up so that we could write concise `{}`-dicts in the param statements, then convert them to `OrderedDict` via `OrderedDict(list(sorted(xxx.items())))` below. I think this is still nice to do for readability.

Please let me know if I've misunderstand some issue with pytest collection and the `OrderedDict` change is required.

Reply:
The `OrderedDict` changes to test parameters in this PR are required for using `pytest-xdist`.  Otherwise, test collection will fail with multiple workers.

Reply:
Thanks for explaining! I'll avoid this pattern in the future.